assemblies/plugins/pom.xml

```diff
@@ -680,6 +681,18 @@
             </exclusion>
           </exclusions>
         </dependency>
+        <dependency>
```

Comment:
This dependency should be up in the main group of dependencies, not inside any of the profiles.

Reply:
addressed review comments